testing/k8s-binder-k8s-hub/binderhub-chart-config.yaml

```diff
@@ -1,8 +1,5 @@
 # This config is used when both BinderHub and the JupyterHub it uses are
 # deployed to a kubernetes cluster.
-# note: when changing the config schema,
-# the old version of this file may need to be copied to ./binderhub-chart-config-old.yaml
-# before updating, and then deleted in a subsequent PR.
```

Comment:
How will future schema breaks be handled?

Reply:
Marking this as draft to think a bit further about this.

Reply:
Related:
- https://github.com/jupyterhub/binderhub/pull/1472#issuecomment-1108426918
- https://github.com/jupyterhub/binderhub/pull/1472#issuecomment-1108640744